src/api/routes/meetings.ts

```diff
@@ -118,6 +119,11 @@ export async function updateOngoingMeetings() {
       attributes: ["tmUserId", "meetingId"], lock: true, transaction: t,
     });
     for (const meeting of ongoingMeetings) {
+      // This condition is added to improve the corner case when a user created a meeting but did not join
+      // If the user did not join meeting right away, other users of the same group might create a new meeting
+      // and overwrite the link. A 3-min grace period is added after a meeting is created to prevent such case.
+      if (moment().diff(meeting.createdAt, 'minutes') < 3) {continue;};
```

Comment:
`{continue;};` => `continue;`

and I'm surprised the ts compiler didn't complain because there is no space between words and brackets.